PWGHF/Core/HfMlResponseDplusToPiKPi.h

```diff
@@ -0,0 +1,162 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HfMlResponseDplusToPiKPi.h
+/// \brief Class to compute the ML response for D± → π± K∓ π± analysis selections
+/// \author Alexandre Bigot <alexandre.bigot@cern.ch>, IPHC Strasbourg
+
+#ifndef PWGHF_CORE_HFMLRESPONSEDPLUSTOPIKPI_H_
+#define PWGHF_CORE_HFMLRESPONSEDPLUSTOPIKPI_H_
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "PWGHF/Core/HfMlResponse.h"
+
+namespace o2::analysis
+{
+
+enum InputFeaturesDplusToPiKPi : int8_t {
+  ptProng0 = 0,
+  ptProng1,
```

Comment:
Hi @vkucera , thanks for the comments.

No, we don't need it to be the same name as the getters. However, this `enum` will be as a dictionary: it will impose the naming convention to use in JSON.

Reply:
You mean these would be the labels of the array that goes in `hfMlResponse.configure(binsPtMl, cutsMl, cutDirMl, nClassesMl);`?

Reply:
It would be the labels used in the configurable vector of strings `namesInputFeatures` called here `hfMlResponse.cacheInputFeaturesIndices(namesInputFeatures);`.

Reply:
OK, got it. Thanks!